fms/modules/attention.py

```diff
@@ -593,6 +661,9 @@ def __init__(
             linear_config=linear_config,
         )
 
+        self.sinks = nn.Parameter(torch.empty(self.nheads))
```

Comment:
this should be gated behind self.has_sinks, otherwise it's going to change every single model in FMS